src/Table/README.md

```diff
@@ -44,3 +45,6 @@ Specifies the screen reader only text that accompanies the sort buttons for sort
 3. `defaultText`: (string) specifies the screen reader only text for sort buttons that are not engaged.
 
 It is only required if `tableSortable` is set to `true`.
+
+### `tableFixed` (boolean; optional)
```

Comment:
I know this isn't the case for the other `props` in this README but I think it would be valuable to add the default prop value?